lib/modules/versioning/gradle/index.ts

```diff
@@ -217,7 +218,7 @@ function getNewValue({
     }
   }
 
-  return null;
+  return mavenVersion.getNewValue({ currentValue, rangeStrategy, newVersion });
```

Comment:
What happens here if the currentValue or newVersion aren't valid maven syntax - can that happen?

Reply:
gradle supports all valid maven versions, any invalid maven version should simply return null